tutorials/image/cifar10/cifar10.py

```diff
@@ -162,6 +165,33 @@ def inputs(eval_data):
   return images, labels
 
 
+def maybe_download_and_extract():
```

Comment:
This seems new.  There is already a download and extract script I think.  Can you explain?

Reply:
You are correct there is a better way to get the data using tensorflow_datasets.
And there are more functions missing. I will fix it.
But currently as it is tutorials/image/cifar10/cifar10_multi_gpu_train.py wont run because of 
function call on line 269, it calls non existent `maybe_download_and_extract`
[here](https://github.com/tensorflow/models/blob/a68f65f8b06969ee314354e3692fce10be7712c7/tutorials/image/cifar10/cifar10_multi_gpu_train.py#L269)